lib/endpoint.ex

```diff
@@ -17,7 +17,7 @@ defmodule AshJsonApiWrapper.Endpoint do
   def schema do
     [
       action: [
-        type: {:wrap_list, :atom},
```

Comment:
I think we might want this to be allowed to be a list and change the matching logic to check if the action is in that list.

Reply:
Sure.

Note: I have left the comparison at:

https://github.com/ash-project/ash_json_api_wrapper/blob/135c5edf1bb5da80e0ba019cdceb474dd1141680/lib/data_layer/transformers/set_endpoint_defaults.ex#L28

unchanged. It seems to be comparing the entire list and I was not able to get my head around it enough to write a failing test

Reply:
Ah, good call, that needs to be switched to use the new identifier logic. I'm leaving this comment here for myself later. Will get this merged in the next day or two.